lib/whitehall_importer/create_revision.rb

```diff
@@ -16,9 +16,10 @@ def self.call(*args)
       new(*args).call
     end
 
-    def initialize(document_import, whitehall_edition)
+    def initialize(document_import, whitehall_edition, change_history = [])
       @document_import = document_import
       @whitehall_edition = whitehall_edition
+      @change_history = change_history
```

Comment:
Does seem a little bit of a bummer that we use the same variable name here and in create_edition to refer to different objects 🤔 I can't think of a nice solution as I think the name we have in here is correct and the most important one, nor do I really want to suggest making the other one longer like change_history_extractor or something. I guess it'll all be ok

Reply:
I know I felt the same when I added this but came to the same conclusion as you. 🤷‍♂